dns/pfSense-pkg-bind9/files/usr/local/pkg/bind.inc

```diff
@@ -163,6 +179,13 @@ options {
 	max-cache-size {$ram_limit};
 
 EOD;
+	if (count($rpz_zones) > 0) {
+		$bind_conf .= "\tresponse-policy {\n";
+		foreach ($rpz_zones as $rpz) {
+			$bind_conf .= "\t\tzone \"$rpz\";\n";
```

Comment:
Variables embedded in strings must be enclosed in braces, for example:
`$bind_conf .= "\t\tzone \"{$rpz}\";\n";`